### PR TITLE
Issue #4: Fix PEP8 on 6.1 branch

### PR DESCRIPTION
**Pull request by mv-organizeit**

### CODE REVIEW

File: hr_timesheet_fulfill/wizard/timesheet_fulfill.py [hunk 1]
```diff
@@ -76,10 +76,10 @@ def fulfill_timesheet(self, cr, uid, ids, context):
 
         # Get the employee
         employee_id = employee_obj.search(cr, uid,
-                [('user_id', '=', uid)], context=context)[0]
+                                          [('user_id', '=', uid)], context=context)[0]
```
Comment (mdietrichc2c) [inline hr_timesheet_fulfill/wizard/timesheet_fulfill.py]: Isn't this line > 79 characters?

File: hr_timesheet_fulfill/wizard/timesheet_fulfill.py [hunk 2]
```diff
@@ -54,11 +54,11 @@ class FulfillTimesheet(osv.osv_memory):
...
         'analytic_account_id': fields.many2one('account.analytic.account',
-                               'Analytic Account', required=True,
-                               domain="[('type', '=', 'normal'),"
-                                      "('state', '!=', 'pending'),"
-                                      "('state', '!=', 'close')]"),
-        'task_id':fields.many2one('project.task','Task', required=False)
+                                               'Analytic Account', required=True,
+                                               domain="[('type', '=', 'normal'),"
+                                                      "('state', '!=', 'pending'),"
+                                                      "('state', '!=', 'close')]"),
```
Comment (mdietrichc2c) [inline hr_timesheet_fulfill/wizard/timesheet_fulfill.py]: Those lines are > 80 characters.

File: hr_timesheet_holidays/company.py [hunk 3]
```diff
@@ -32,16 +32,16 @@
...
-        'timesheet_hours_per_day': fields.float('Timesheet Hours Per Day', digits=(2,2))
+        'timesheet_hours_per_day': fields.float('Timesheet Hours Per Day', digits=(2, 2))
     }
 
     _defaults = {
         'timesheet_hours_per_day': 8.0
     }
 
-res_company()
+ResCompany()
```
Comment (StefanRijnhart) [inline hr_timesheet_holidays/company.py]: Initialization is not necessary anymore since Odoo 6.1

**Comment by mdietrichc2c** [comment]
Tested on https://travis-ci.org/taktik/hr-timesheet/builds/65068841 , checks as green. :+1: barring any blocking issues.

**Comment by StefanRijnhart** [comment]
The code on which I commented is now outdated, but the nag still stands: please remove all the initialization calls that you rewrote, like

```
ResCompany()
```

**Comment by mv-organizeit** [comment]
I've remove the nag. Don't understand why runbot failed, i try to install the module and I have no error

**Comment by mdietrichc2c** [comment]
FYI: the runbot fails because it's trying to deploy a 8.0 version.

**Comment by gurneyalex** [comment]
CLA ok

**Comment by pedrobaeza** [comment]
:+1:

**Comment by StefanRijnhart** [comment]
Thanks for the update! :+1:

**Comment by mv-organizeit** [comment]
Thanks for the merge! :+1:

**Pull request merged**